provision.sh

```diff
@@ -14,8 +14,8 @@ sudo dnf versionlock add qemu-kvm-2.12.0-88.module+el8.1.0+5708+85d8e057.3
 sudo dnf install -y libvirt libvirt-devel libvirt-client git libvirt-daemon-kvm bind-utils jq gcc-c++
 
 # Install golang
-curl -L https://dl.google.com/go/go1.13.8.linux-amd64.tar.gz -o go1.13.8.linux-amd64.tar.gz
-tar -xvf go1.13.8.linux-amd64.tar.gz
+curl -L https://dl.google.com/go/go1.15.8.linux-amd64.tar.gz -o go1.15.8.linux-amd64.tar.gz
+tar -xvf go1.15.8.linux-amd64.tar.gz
```

Comment:
Why can't we use `golang` from offical repo , it is `golang-1.14.12-1.module+el8.3.0+8784+380394dc.x86_64` as of now. Are we trying to build something on this VM which require latest golang?

Reply:
kube uses  go 1.15 as of v 1.19, so keeping in sync with rest of kube/openshift

Reply:
Hrm, would be nice to get RHEL `go-toolset` updated to 1.15 if that's a requirement for openshift builds.

Reply:
@sallyom but we are not compiling the kube/openshift part on these VM for test right?

Reply:
no, we aren't so official repo golang should be fine.